Core/run.mac

```diff
@@ -22,25 +22,25 @@
 /eAST/field/central/type cubic
 /eAST/field/central/load data/EIC_Magnetic_Field_Map_2021_05_07_radial_coords__cm___T_.120000.lines.Bmap
 
-/eAST/field/print (0.0, 0.0, 0.0)
-/eAST/field/print (1000.0, 0.0, 0.0)
-/eAST/field/print (0.0, 1000.0, 0.0)
-/eAST/field/print (0.0, 0.0, 1000.0)
-/eAST/field/print (0.0, 1000.0, 1000.0)
-/eAST/field/print (1000.0, 0.0, 1000.0)
-/eAST/field/print (1000.0, 1000.0, 0.0)
-/eAST/field/print (1000.0, 1000.0, 1000.0)
-/eAST/field/print (0.0, 0.0, 10000.0)
+# /eAST/field/print (0.0, 0.0, 0.0)
+# /eAST/field/print (1000.0, 0.0, 0.0)
+# /eAST/field/print (0.0, 1000.0, 0.0)
+# /eAST/field/print (0.0, 0.0, 1000.0)
+# /eAST/field/print (0.0, 1000.0, 1000.0)
+# /eAST/field/print (1000.0, 0.0, 1000.0)
+# /eAST/field/print (1000.0, 1000.0, 0.0)
+# /eAST/field/print (1000.0, 1000.0, 1000.0)
+# /eAST/field/print (0.0, 0.0, 10000.0)
 
 #######################################################
 # Detector components
 #######################################################
-/eAST/component/beampipe 1
-/eAST/component/beampipe/envelopeGdmlFile ../Components/Beampipe/data/Detector_chamber_210222_BG-05-21-2021_single_volume.gdml
-/eAST/component/beampipe/gdmlFile ../Components/Beampipe/data/Detector_chamber_210222_test1_BG-04-07-2021.gdml
-/eAST/component/beampipe/materialFile ../Components/Beampipe/data/Detector_chamber_210222_test1_BG-04-07-2021.mat
-/eAST/component/beampipe/rotate x -90. deg
-/eAST/component/beampipe/rotate y -90. deg
+# /eAST/component/beampipe 1
```

Comment:
Shouldn't we keep this enabled? Even if we know it doesn't work on your setup?

Reply:
Fair question, I wasn't sure what to do. I don't want a user to try and test the hepmc interface and than run into obscure and mostly unrelated problems. But I'll reenable it and add some comments in the macro.